data/seeds/virtualParent.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('VirtualParent').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('VirtualParent').insert([
+        {id: 1, email: '', first_name:"Anne",last_name:"McVirtual"},
+        {id: 2, email: '', first_name:"Randy",last_name:"McVirtual"},
+        {id: 3, email: '', first_name:"Bob",last_name:"McVirtual"}
```

Comment:
Let's replace the contents of this file with what you added in the parents seed file, then revert the parents seed file back to how it was before. Also, try to structure this file like how the parents seed file was structured 👍